app/src/main/java/org/fossasia/openevent/general/event/EventViewHolder.kt

```diff
@@ -61,17 +61,16 @@ class EventViewHolder(override val containerView: View) : RecyclerView.ViewHolde
 
         setFabBackground(event.favorite)
 
-        event.eventType.let {
-            if (it != null)
+        if (containerView.chipTags != null) {
+            event.eventType?.let {
                 addchips(it.name)
-        }
-        event.eventTopic.let {
-            if (it != null)
+            }
+            event.eventTopic?.let {
                 addchips(it.name)
```

Comment:
Wrong indentation

Reply:
@iamareebjamal Klint test would have failed if the indentation was wrong